sources/packages/backend/apps/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -141,3 +145,12 @@ export class SearchInstitutionAPIOutDTO {
   operatingName: string;
   address: AddressAPIOutDTO;
 }
+
+export class SearchInstitutionQueryAPIInDTO {
```

Comment:
Not sure if it is part of this PR, but I would assume that for this search one or another parameter must be provided, right? 
For these cases, we use the `@ValidateIf` to enforce that one of them must be provided. There are some examples in the code already, for instance `AESTStudentSearchAPIInDTO`.